Automattic/jetpack
# AI Proofread: Get and apply suggestions

Get and apply suggestions.

ONLY working with Complex Words.
NEEDS: D155332-code

## Proposed changes:

* Add UI for suggestion.
* Add handling suggestion.
* Saving suggestion on store.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Make sure to sandbox your WPCOM.
* Go to your editor.
* Add some post that has Complex Words.
* Ask for Suggest.
* It should suggest.
* Accept it.
* It should replace the word.
